File: cfgs/pythoncfgs/FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES.py
Nodes
1;statement;foo_1 = [0] * (foo_2 + 1);8;
2;statement;foo_1[0] = 0;9;
3;statement;foo_1[1] = 1;10;
4;conditional;foo < foo;11;
5;statement;return foo_1[foo_2];16;
6;conditional;fun_1(foo_1 % 2) == 0;12;
7;statement;foo_1[foo_2] = foo_3[fun_1((foo_4 - 1) / 2)] + foo_5[fun_2((foo_6 + 1) / 2)];15;
8;statement;foo_1[foo_2] = foo_3[fun_1(foo_4 / 2)];13;
9;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;9;
5;7;8;
6;4;
7;4;
8;

File: cfgs/java_cfgs_foo/FIND_N_TH_ELEMENT_FROM_STERNS_DIATOMIC_SERIES.java
Nodes
1;special;;;
2;statement;int[] DP = new int[foo_1 + 1];14;
3;statement;foo_1[0] = 0;15;
4;statement;foo_1[1] = 1;16;
5;statement;int i = 2;17;
6;conditional;foo_1 <= foo_2;18;
7;conditional;(foo_1 % 2) == 0;20;
8;statement;foo_1[foo_2] = foo_3[foo_4 / 2];20;
9;statement;foo_1[foo_2] = foo_3[(foo_4 - 1) / 2] + foo_5[(foo_6 + 1) / 2];21;
10;statement;i++;19;
11;statement;return foo_1[foo_2];23;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;11;
6;8;9;
7;10;
8;10;
9;6;
10;12;
11;
